native/acter/src/api/room.rs

```diff
@@ -156,7 +156,7 @@ impl Member {
         let tester: PermissionTest = match permission {
             MemberPermission::CanBan => return self.member.can_ban(),
             MemberPermission::CanInvite => return self.member.can_invite(),
-            MemberPermission::CanRedact => return self.member.can_redact(),
+            MemberPermission::CanRedact => return self.member.can_redact_own(),
```

Comment:
I don't think this change is correct. This would not allow admins to redact items anymore more. `can_redact_other()` is also relevant here. we might have to refactor the entire way this works ... at least also put up these two different modes of redaction up: split the current one into `MemberPermission::CanRedactOwn` and `MemberPermission::CanRedactOther` ...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1311/commits/1989a5e1c1f64d6b0d0d0761ed32291247856c59